core_weekly/github.py

```diff
@@ -1,37 +1,68 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
+from github import Github
+from github import UnknownObjectException
 import requests_cache
 import logging
 import requests
 import ssl
 import time
+from datetime import date, datetime, timedelta
 
 logger = logging.getLogger(__name__)
 ssl._create_default_https_context = ssl._create_unverified_context
 
 
-# Github api
+# Github API Client
 #
-class GitHub():
+class GitHubClient():
     retries = 0
 
-    def __init__(self, no_cache, debug):
+    def __init__(self, no_cache, debug, github_access_token):
         """Constructor
 
         :param no_cache: Disable cache
         :type no_cache: bool
         :param debug: Is debug mode enabled
         :type debug: bool
+        :param github_access_token: GitHub Access Token
+        :type github_access_token: str
 
         """
         self.sleep_time = 3
         self.url = 'https://api.github.com/search/issues?per_page=100'
         self.no_cache = no_cache
         self.is_debug = debug
+        self.github_access_token = github_access_token
 
         if not self.no_cache:
             requests_cache.install_cache('cache')
 
+    def get_last_week_git_releases(self):
+        """Get last week GitRelease objects (see https://pygithub.readthedocs.io/)
+
+        :returns: List of dictionaries <GitRelease, Repository>
```

Comment:
I dont know how to describe dictionary content in Python

Reply:
You can do this [typing](https://docs.python.org/3/library/typing.html):

```python
from typing import Mapping

def func() -> Mapping[str, int]
    """Some function to do something to a dictionary."""
    ...
```